react-app/src/components/UI/RecursiveComponent.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from '../../styles/RecursiveComponent.module.css';
+
+export default function RecursiveComponent({ message, depth, maxDepth }: { message: string, depth: number, maxDepth: number }) {
```

Comment:
On vue branch, you have type for this component, make it consistent

Reply:
fixed inconsistency